robot_skills/package.xml

```diff
@@ -37,6 +37,7 @@
   <depend>tue_manipulation_msgs</depend>
   <depend>tue_msgs</depend>
   <depend>visualization_msgs</depend>
+  <depend>tmc_suction</depend>
```

Comment:
It kinda sucks that I need to include a toyota specific package here. Because a robot that doesn't use suction cup arms doesnt need this dependency.

Reply:
Indeed, this should be handled differntly.

Reply:
But how? The suction cup interface needs this message type. The only way I see this happening is if we are able to separate  each robot part in a separate repo so that the robot can be composed only of the parts that it has and use the dependencies of its parts only.

Reply:
I also don't like having this dependency, but I don't see any other way either. One way or another, we'll need those messages.

An option could be to have a separate package with a more general (non-TMC) suction cup interface, but for TMC that would still have to depend on `tmc_suction`. That conversion could be done in a bridge node, specific for HERO. That is a bit more complex due to the indirection.